.eleventy/docs/chainlink-automation/supported-networks.html

```diff
@@ -4,43 +4,44 @@
 date: Last Modified
 title: 'Supported Blockchain Networks'
 whatsnext:
-  {
-    'Automation Economics': '/docs/chainlink-automation/automation-economics/',
-    'FAQs': '/docs/chainlink-automation/faqs/',
-  }
+{
+'Automation Economics': '/docs/chainlink-automation/automation-economics/',
+'FAQs': '/docs/chainlink-automation/faqs/',
+}
 networks:
-  {
-    'Ethereum': 'ethereum',
-    'Polygon': 'polygon',
-    'BNB Chain': 'bnbchain',
-    'Avalanche': 'avalanche',
-    'Fantom': 'fantom',
-  }
+{
+'Ethereum': 'ethereum',
+'Polygon': 'polygon',
+'BNB Chain': 'bnbchain',
+'Avalanche': 'avalanche',
+'Fantom': 'fantom',
+}
 ---
 
 {% markdown %}
 
 Chainlink Automation is available on the following networks:
 
 - [Ethereum](#ethereum):
-  - Mainnet
-  - Goerli testnet
+- Mainnet
+- Goerli testnet
 - [Polygon (Matic)](#polygon):
-  - Mainnet
-  - Mumbai testnet
+- Mainnet
+- Mumbai testnet
 - [BNB Chain](#bnb-chain):
-  - Mainnet
-  - Testnet
+- Mainnet
+- Testnet
 - [Avalanche](#avalanche):
-  - Mainnet
-  - Fuji testnet
+- Mainnet
+- Fuji testnet
 - [Fantom](#fantom):
-  - Mainnet
-  - Testnet
+- Mainnet
+- Testnet
 - [Arbitrum](#arbitrum)
-  - Mainnet
+- Mainnet
 
-If you are interested in using Chainlink Automation but need it on a different blockchain or network, [contact us](https://forms.gle/WadxnzzjHPtta5Zd9) and let us know.
+If you are interested in using Chainlink Automation but need it on a different blockchain or network, [contact
```

Comment:
I took another look at this file and I don't think there are any actual changes, aside from spacing changes likely introduced by Prettier. If you didn't change this file intentionally, I suggest reverting the changes to the file and adding a prettier-ignore statement ahead of any tables - or simply dropping this file from the PR, whichever is easier for you!